pandas/tests/test_index.py

```diff
@@ -133,6 +133,14 @@ def test_str(self):
         self.assertTrue("'foo'" in str(idx))
         self.assertTrue(idx.__class__.__name__ in str(idx))
 
+    def test_repr_max_seq_item_setting(self):
```

Comment:
ahh, you know what the function above it is not testing much ``test_str`` (that's what I was looking at). should be iterating over the ``self.indices`` rather than using ``.create_index()``.